source/OptimaJet.DWKit.StarterApplication/Templates/ReviewProduct.txt

```diff
@@ -0,0 +1,9 @@
+﻿@{
+    DisableEncoding = true;
+}
+<html>
```

Comment:
We are going to have to figure out what looks good as an HTML email.  Maybe there are libraries???